homeassistant/components/blinksticklight/light.py

```diff
@@ -16,13 +16,18 @@
 
 CONF_SERIAL = "serial"
 
-DEFAULT_NAME = "Blinkstick"
+CONF_INDEX = "index"
+
+DEFAULT_NAME = "Blinkstick Square"
+
+DEFAULT_INDEX = 0
 
 SUPPORT_BLINKSTICK = SUPPORT_BRIGHTNESS | SUPPORT_COLOR
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_SERIAL): cv.string,
+        vol.Optional(CONF_INDEX, default=DEFAULT_INDEX): cv.positive_int,
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>